networktable.c

```diff
@@ -16,75 +16,84 @@
 static char* getMACVendor(const char* hwaddr);
 bool unify_hwaddr(sqlite3 *db);
 
+// Private macro
+#define SQL_bool(sql) {\
+	if(!dbquery(sql)) {\
+		logg("%s(): \"%s\" failed!", __FUNCTION__, sql);\
+		dbclose();\
+		return false;\
+	}\
+}
+#define SQL_void(sql) {\
+	if(!dbquery(sql)) {\
+		logg("%s(): \"%s\" failed!", __FUNCTION__, sql);\
+		dbclose();\
+		return;\
+	}\
+}
```

Comment:
These can be moved to `routines.h`, as they can be used by anything that interacts with the database.